source/documentation/guidance/buildpacks.md

```diff
@@ -1,44 +1,120 @@
-
-Using a custom buildpack will mean increased development and maintenance responsibilities and costs for you. If your language is not supported by standard buildpacks, we recommend that for a community supported custom buildpack that is maintained and updated by that community. We do not recommend that you create your own buildpack.
-
-We are keen to help where we can, so please contact us if you are interested in this feature. If a custom buildpack is requested by multiple tenants, we will look to improving our support in this area.
-
-### Responsibilities
-
-Your responsibilities differ depending on whether you use a standard buildpack, custom buildpack or Docker image to deploy your app.
+## Responsibility model
+
+Your responsibilities differ depending on whether you use a [standard buildpack](/#buildpacks), [custom buildpack](/#how-to-use-custom-buildpacks) or [Docker image](/#deploy-a-docker-image-experimental) to deploy your app.
+
+<div class="table-container">
+	<table>
+		<tbody>
+			<tr>
+				<th style="vertical-align: middle; text-align: left">Responsibility</th>
+				<th style="vertical-align: middle; text-align: center">Standard Buildpack</th>
+				<th style="vertical-align: middle; text-align: center">Custom Buildpack</th>
+				<th style="vertical-align: middle; text-align: center">Docker Images</th>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Your app
+					<br> - apply security updates to app dependencies
+					<br> - run <a href="https://docs.cloud.service.gov.uk/#penetration-testing">pen tests</a>
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Language / runtime
+					<br> - update language and runtime
+          <br> - provide consistent app build process
+					<br> - monitor and patch runtime vulnerabilities
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Base operating system
+					<br> - update OS libraries
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					App admin
```

Comment:
`App admin` sounds strange and the tenants may not understand what we mean by that. Use `Application platform` instead.
Fort the list I suggest:
 - ensure high availability
 - ensure provisioning capacity
 - monitoring

I'm not sure `stream application logs` has any information or value here.

Reply:
I wouldn't put *monitoring* because we do not do any monitoring at all.

Reply:
Instead of *Application platform* , what about *Application lifecycle*

> Application lifecycle
>  * ensure high availability
>  * ensure provisioning capacity
>  * stream application logs and metrics

Reply:
This is the Cloud Foundry layer itself, so I would call it platform and not lifecycle. And we do monitoring there, e.g. if your app dies then we restart it automatically.

Monitoring the apps is a responsibility.